ci/ci-iwyu.py

```
@@ -416,6 +417,133 @@ def check_iwyu_available() -> tuple[bool, str]:
+    jobs = str(os.cpu_count() or 4)
+    cmd: list[str] = [
+        "uv",
+        "run",
+        "clang-tool-chain-iwyu-tool",
+        "-p",
+        str(compile_db.parent),
+        "-j",
+        jobs,
+        *files,
+        "--",
+    ] + iwyu_tail
```

Ruff (0.15.10)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)